recipes/protoc-gen-swagger/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "protoc-gen-swagger" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Only uploaded to pypi as a .whl, not .sdist, so package is specified as github source archive
+  url: https://github.com/universe-proton/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

```suggestion
  url: https://github.com/universe-proton/protoc-gen-swagger/archive/refs/tags/v{{ version }}.tar.gz
```

Extra variables here can sometimes trip up the bot.

---

Thanks!